libr/include/r_io.h

```diff
@@ -393,6 +393,10 @@ R_API void r_io_desc_free (RIODesc *desc);
 R_API bool r_io_desc_add (RIO *io, RIODesc *desc);
 R_API bool r_io_desc_del (RIO *io, int fd);
 R_API RIODesc *r_io_desc_get (RIO *io, int fd);
+R_API RIODesc *r_io_desc_get_next (RIO *io, RIODesc *desc);
```

Comment:
functin definitions shoudlnt have a space before (

Reply:
this issue is not part of this pr, imo there should be a dedicated pr to fix that file